src/robot_start_stop.py

```
@@ -0,0 +1,31 @@
+from flask import Blueprint, request, jsonify
+import subprocess
+
+blueprint = Blueprint('robot_start_stop', __name__)
+
+@blueprint.route('/start', methods=['POST'])
+def claim_and_start_robot():
+    user = request.form['user']
+
+    subprocess.check_call('robot claim -f --username ' + user + ' --email ' + user + ' -m "teleoperating the robot (claimed via Robot Web Server)"', shell=True)
+    subprocess.check_call('robot start -f', shell=True)
+
+    return jsonify({'status': 'success'})
```

Maybe return an error value based on the result of check_call?